invenio_app_ils/circulation/receivers.py

```diff
@@ -53,6 +53,11 @@ def index_after_loan_replace_item(_, old_item_pid, new_item_pid):
 
 def send_email_after_loan_change(_, initial_loan, loan, trigger):
     """Send email notification when the loan changes."""
+    cancel_reason = loan.get("cancel_reason", None)
+    if cancel_reason == "Anonymization.":
```

Comment:
I am sorry but I am still wondering if we really need this.
Since the `patron` is `-2`, why this?

Reply:
I answered above https://github.com/inveniosoftware/invenio-app-ils/pull/1037#discussion_r583677304

Reply:
Thanks for the explanations. As discussed, let's remove this if and let's send the email.